src/components/LowerLayout/GeneralInfo/index.js

```diff
@@ -10,23 +9,26 @@ const createList = listContainer => listContainer.map(({ list }) => (
 ));
 
 const createLink = additionalInfo => additionalInfo.map(({ link }) => {
-  if (!link) {
-    return null;
+  if (link) {
+    return (
+      <a
+        className={classes.link}
+        key={link}
+        href="https://search.opengazettes.org.za/text/25860?page=31"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {link}
+      </a>
+    );
   }
-
-  return (
-    <a key={link}
-      href="https://search.opengazettes.org.za/text/25860?page=31"
-      target="_blank"
-      rel="noopener noreferrer"
-    >{link}</a>
-  );
 });
 
-const createAdditionalInfo = additionalInfo => additionalInfo.map(({ info }) => (
-  <p key={info}>
+const createAdditionalInfo = additionalInfo => additionalInfo.map(({ info, link }) => (
+  <p
```

Comment:
Just make this one line, usually split it to more than one line if 3 or more attributes.

Reply:
do you mean like this? 
`const createAdditionalInfo = additionalInfo => additionalInfo.map(({ info, link }) => (
  <p key={info}> {info} {link && createLink(additionalInfo)}</p>
));`

Reply:
`const createAdditionalInfo = additionalInfo => additionalInfo.map(({ info, link }) => (
  <p key={info}> {info} {link && createLink(additionalInfo)}</p>
));`

Reply:
like this?